packages/ui/src/screens/home/components/hero/components/online_voting_power/index.tsx

```diff
@@ -11,7 +11,7 @@ const OnlineVotingPower: FC<ComponentDefault> = () => {
 
   const votingPowerPercent =
     state.totalVotingPower === 0
-      ? numeral(0)
+      ? numeral((state.activeValidators / state.votingPower) * 100)
```

Comment:
can we verify for a divide by zero error here?

Reply:
could you clarify? @teamchong

Reply:
my initial impression is that if the state.votingPower is zero, a division by zero error will be raised.

Reply:
so the feature was made for wormhole, and the voting power and active validator set returned from wormhole backend was null 0. But I do see that there might be edge cases in the future, so I have pushed a commit to address your comments, please review @teamchong